### ISSUE TITLE
Add search

### ISSUE THREAD

**Issue by janosimas** [spmeesseman/vscode-taskexplorer#140]
I'm working on a project composed of many small sub-projects, in the end I have hundreds of tasks.
Having a search function would be amazing.

**Comment by janosimas** [comment]
I was thinking, maybe a filter field.
This way the tasks could be filtered by part of the name of the sub-folder.

**Comment by spmeesseman** [comment]
good idea about a filter but i dont think the vscode tree provider supports a textfield within the control, it would have to be a command pallet command to filter (or search), which would be kinda crappy, not sure.  what do you think

**Comment by janosimas** [comment]
I agree it would be crappy but if there is no option...
It could have a button on top to fav and it would open a search box where you could add to the favorites.
